rolling_update.yml

```diff
@@ -59,7 +68,7 @@
       until: result.rc == 0
       retries: 5
       delay: 10
-      delegate_to: 127.0.0.1
+      delegate_to: "{{ mon_host }}"
```

Comment:
Sorry to be picky but assuming I have 3 monitors, this variable will contain 2 monitors right?
So we actually perform this action on 2 mons, this is not really an issue, but ideally we would pick the first one of this list as well.